FormConfigurationFactory.php

```diff
@@ -88,6 +90,14 @@ private function getFieldsConfiguration($model, $fields = [])
             if (isset($fields[$property->getName()])) {
                 $configuration[$property->getName()] = array_replace_recursive($configuration[$property->getName()], $fields[$property->getName()]);
             }
+            if ($configuration[$property->getName()]['type'] === EmbedType::TYPE) {
+                if (($value = $property->getValue($model)) === null) {
+                    $instantiator = new Instantiator();
+                    $value = $instantiator->instantiate($configuration[$property->getName()]['class']);
+                }
+                $configuration[$property->getName()]['data_class'] = $configuration[$property->getName()]['class'];
```

Comment:
Is there any point in introducing `class` property instead of relying on Symfony's `data_class`? TBH I like "class" better, just wondering :)